### PR TITLE
add bert-small-uncased-whole-word-masking-squad-0002 model with reduced size

### PR DESCRIPTION
**Pull request by krodyush**
number of parameters is reduced from 57.9M to 41.1M in comparison to bert-small-uncased-whole-word-masking-squad-0001 model
@myshevts

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

File: models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+`[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` + (`[PAD]` tokens to pad to the maximum sequence length of 384)
+
+2. Input mask, name: `attention_mask`, shape: [1x384].
+Input mask is a sequence of integer values representing the mask of valid values in the input.
+The values of this input are equal to:
+    * `1` at positions corresponding to the `[CLS]` + *tokenized question* + `[SEP]` + *tokenized premise of the question* + `[SEP]` part of the `input_ids`  (i.e. all positions except those containing the `[PAD]` tokens), and
+    * `0` at all other positions
+
+3. Token types,  name: `token_type_ids`, shape: [1x384].
+Token types is sequence of integer values representing the segmentation of the `input_ids` into question and premise.
```
Comment (myshevts) [inline models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md:50]: 'premise'? maybe 'context'?

File: models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+The values are equal to:
+    * `1` at positions corresponding to the *tokenized premise of the question* + `[SEP]` part of the `input_ids`, and
+    * `0` at all other positions
+
+* `[CLS]` is a special symbol added in front of the question.
+* `[SEP]` is a special separator token inserted between the question and premise of the question.
+* `[PAD]` is a special token used to fill the rest of the input to get given input length (384).
+
+4. Position indexes,  name: `position_ids`, shape: [1x384].
+Position indexes is sequence of integer values from 0 to 383 representing the position index for each input token. The `position_ids` is lways the same for any input tokens set
```
Comment (myshevts) [inline models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md:60]: lways

File: models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md (hunk 1 above)
Comment (krodyush) [reply models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md:50]: I agree to use any of them but this word (premise) is used in description of all other bert models. if I change it here it will be inconsistent. should we change it in each bert model description? 
add @vshampor to discuss this.

File: models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md (hunk 2 above)
Comment (krodyush) [reply models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md]: Thanks!

File: models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md (hunk 1 above)
Comment (vshampor) [reply models/intel/bert-small-uncased-whole-word-masking-squad-0002/description/bert-small-uncased-whole-word-masking-squad-0002.md:50]: https://dictionary.cambridge.org/dictionary/english/premise

Personally I'm ok with "context" as well, as long as it is also used consistently across all descriptions.

File: tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py [hunk 3]
```diff
@@ -255,14 +255,18 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
...
+                              'segment_ids_{}'.format(idx),
+                              'position_ids_{}'.format(idx)]
                 annotation = QuestionAnsweringAnnotation(
                     identifier,
                     example.qas_id,
                     np.array(unique_id),
                     np.array(span["input_ids"]),
                     np.array(span["attention_mask"]),
                     np.array(span["token_type_ids"]),
+                    np.arange(len(span["token_type_ids"])),#position_ids
```
Comment (IRDonch) [inline tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py:269]: Instead of a comment, I would suggest to use a named argument.

File: models/intel/bert-small-uncased-whole-word-masking-squad-0002/accuracy-check.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            type: INPUT
+            value: 'segment_ids'
+          - name: 'position_ids'
+            type: INPUT
+            value: 'position_ids'
+
+    datasets:
+      - name: squad_v1_1_msl384_mql64_ds128_lowercase
+        annotation_conversion:
+          vocab_file: vocab.txt
```
Comment (eaidova) [inline models/intel/bert-small-uncased-whole-word-masking-squad-0002/accuracy-check.yml:26]: looks confusing, please move this part to dataset_definitions.yml

Comment (krodyush) [reply models/intel/bert-small-uncased-whole-word-masking-squad-0002/accuracy-check.yml]: ok. 
Also I've made the same for previous model models/intel/bert-small-uncased-whole-word-masking-squad-0001

File: tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py (hunk 3 above)
Comment (krodyush) [reply tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py]: ok

**Comment by IRDonch** [comment]
There are some complaints from Pylint - please fix them.
